universal7880-common/universal7880-common-vendor.mk

```diff
@@ -112,10 +123,17 @@ PRODUCT_COPY_FILES += \
     vendor/samsung/universal7880-common/proprietary/lib64/libuniapi.so:system/lib64/libuniapi.so \
     vendor/samsung/universal7880-common/proprietary/lib64/libuniplugin.so:system/lib64/libuniplugin.so \
     vendor/samsung/universal7880-common/proprietary/lib64/libwrappergps.so:system/lib64/libwrappergps.so \
+    vendor/samsung/universal7880-common/proprietary/lib64/libstagefrighthw.so:system/lib64/libstagefrighthw.so \
```

Comment:
This looks out of order. Was the makefile really generated by script?

Reply:
Not really. Right now we are pulling the blobs from three different sources in order to make all the features work properly. We are working on switching to full 'oreo firmware' blobs and that's when we plan to 'start' using script. These commits are cherry-picked from the older 15.1 branch, I'm just trying to merge to 16.0 because I don't want to confuse xda users that want to compile 16.0 from source.